spec/datetimeformat.html

```diff
@@ -131,6 +131,7 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
           1. Else,
             1. Let _value_ be ? GetOption(_options_, _prop_, *"string"*, &laquo; the strings given in the Values column of the row &raquo;, *undefined*).
           1. Set _opt_.[[&lt;_prop_&gt;]] to _value_.
+        1. Set _opt_.[[hourCycle]] to _dateTimeFormat_.[[HourCycle]]. 
```

Comment:
I am in favor of providing this data, but would like to do so in a slightly different way. Let's rename this second _opt_ to something like _formatOptions_ and provide the data in its initialization.
```patch
-        1. Let _opt_ be a new Record.
+        1. Let _formatOptions_ be a new Record.
+        1. Set _formatOptions_.[[hourCycle]] to _dateTimeFormat_.[[HourCycle]].
```

Reply:
done @gibson042